webpack/scenes/Subscriptions/Manifest/ManifestConstants.js

```diff
@@ -14,6 +14,10 @@ export const MANIFEST_HISTORY_REQUEST = 'MANIFEST_HISTORY_REQUEST';
 export const MANIFEST_HISTORY_SUCCESS = 'MANIFEST_HISTORY_SUCCESS';
 export const MANIFEST_HISTORY_FAILURE = 'MANIFEST_HISTORY_FAILURE';
 
+export const MANIFEST_SCA_TOGGLE_REQUEST = 'MANIFEST_SCA_TOGGLE_REQUEST';
```

Comment:
As a new name for these actions (moving away from Toggle) I suggest `SET_SIMPLE_CONTENT_ACCESS_REQUEST` etc.